puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/folderChanged.groovy

```diff
@@ -5,7 +5,8 @@ def folderChanged(folders) {
     files_changed = readFile('files_changed').split('\n')
 
     for (int i = 0; i < folders.size(); i++) {
-        changed = changed || !files_changed.findAll { it =~ folders[i] }.empty
```

Comment:
does findAll {}.empty not work not return false if 0 or found?

Reply:
It seems to be returning an array of values. For example:

```
[false, false, false, false]
```